packages/edit-post/src/components/meta-boxes/use-meta-box-initialization.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * WordPress dependencies
+ */
+import { useDispatch, useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+import { useEffect } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { store as editPostStore } from '../../store';
+
+/**
+ * Initializes WordPress `postboxes` script and the logic for saving meta boxes.
+ *
+ * @param { boolean } enabled
+ */
+export default ( enabled ) => {
```

Comment:
Can we do a named export here, `export default function useMetaBoxInitialization( enabled )`? This would make it easier to search for hooks/components in the codebase.

Reply:
Sure, done in 030c6a5b25ca1b6b1f0e054a48f8258199495a53.